### PR TITLE
Improve auto deploy tag regexp

### PR DESCRIPTION
**Pull request by XVincentX**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ workflows:
       - deploy:
           filters:
             tags:
-              only: /^v\d\.\d\.\d/
+              only: /v[0-9]+(\.[0-9]+)*/
```
Comment (DrMegavolt) [inline .circleci/config.yml]: How about
v\d+(\.\d+){2} 

the problem I see is v1.4 is also valid as well v1

Comment (XVincentX) [reply .circleci/config.yml]: Good idea, this will prevent to make mistakes with tags.

File: .circleci/config.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ workflows:
       - deploy:
           filters:
             tags:
-              only: /^v\d\.\d\.\d/
+              only: /v\d+(.\d+){2}/
```
Comment (DrMegavolt) [inline .circleci/config.yml]: it should be escaped dot `\.\d+` 
or it can be as it was before just with stars 
/^v\d*\.\d*\.\d*/

Comment (DrMegavolt) [reply .circleci/config.yml]: regex101 to test

Comment (XVincentX) [reply .circleci/config.yml]: Gosh! I'll fix it now.

**Comment by XVincentX** [comment]
@DrMegavolt Is it ok now?

**Pull request merged**